### PR TITLE
task 3 - katas

### PR DESCRIPTION
**Pull request by camosu**

### CODE REVIEW

File: camilo-alfonso-tarea3/oddEven.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+/* Given a list of integers, determine whether the sum of its elements is odd or even.
+   Give your answer as a string matching "odd" or "even".
+   If the input array is empty consider it as: [0] (array with a zero). 
+*/
+
+function oddOrEven(array) {
+   let total = array.reduce((prev, curr) => prev+ curr, 0)
```
Comment (emolinaglobant) [inline camilo-alfonso-tarea3/oddEven.js:7]: @camosu , find a way to make this a one-line solution also use more than one return expression is a bad practice

File: camilo-alfonso-tarea3/smallEnough.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+/* You will be given an array and a limit value. You must check that all values in the array are below or equal to the limit value. 
+If they are, return true. Else, return false.
+
+You can assume all values in the array are numbers. */
+
+function smallEnough(a, limit) {
+  let check = a.find((element) => element > limit)
```
Comment (emolinaglobant) [inline camilo-alfonso-tarea3/smallEnough.js:7]: @camosu , find a way to make this a one-line solution also use more than one return expression is a bad practice

File: camilo-alfonso-tarea3/oddEven.js (hunk 1 above)
Comment (camosu) [reply camilo-alfonso-tarea3/oddEven.js:7]: I have already corrected the exercises using ternary operator.

File: camilo-alfonso-tarea3/smallEnough.js (hunk 2 above)
Comment (camosu) [reply camilo-alfonso-tarea3/smallEnough.js:7]: In this case I also use ternary operator to correct the exercise.

**Review by emolinaglobant — APPROVED** [review]

**Pull request merged**